### ISSUE TITLE
Help me with this

### ISSUE THREAD

**Issue by RohitKumar3005** [Krushna153/Multicasting-multimedia-over-IP#1]
receiver: setsockopt() error: No such device

**Comment by seeeooo123** [comment]
hi, i was also getting this error, so when you run ./client [ip-address] put the ip of ur device, which u can find using ipconfig. also make sure the interface in code matches the one on your device which u can also find using ipconfig

**Comment by Krushna153** [comment]
> hi, i was also getting this error, so when you run ./client [ip-address] put the ip of ur device, which u can find using ipconfig. also make sure the interface in code matches the one on your device which u can also find using ipconfig

Yes, that's correct. @RohitKumar3005 try using the IP of your device. I hope this way will resolve your issue.
